hack/util.sh

```diff
@@ -458,10 +458,14 @@ function install_registry() {
 }
 readonly -f install_registry
 
-function wait_for_registry() {
+function is_registry_ready() {
```

Comment:
@stevekuznetsov Where is documentation for this? Why even this file doesn't respect this rules? I see that this file has  28 functions without namespace and only 10 that having it.